Sources/Plasma/Apps/plUruLauncher/winmain.cpp

```diff
@@ -188,6 +189,44 @@ static void PumpMessages()
     } while (s_launcher->PumpNetCore());
 }
 
+void SetDPIAwareness() const
```

Comment:
```suggestion
static void SetDPIAwareness()
```
I know this is temporary code, but this being `const` doesn't make sense because it is not part of a class.

Reply:
Yeah, the `const` modifier is breaking CI:
```
plUruLauncher\winmain.cpp(193,1): error C2270: 'SetDPIAwareness': modifiers not allowed on nonmember functions
```